frontends/mit-open/src/routes.tsx

```diff
@@ -40,16 +42,21 @@ const routes: RouteObject[] = [
         element: <LearningPathDetailsPage />,
       },
       {
-        path: urls.ARTICLES_DETAILS,
-        element: <ArticleDetailsPage />,
-      },
-      {
-        path: urls.ARTICLES_EDIT,
-        element: <ArticleEditingPage />,
-      },
-      {
-        path: urls.ARTICLES_CREATE,
-        element: <ArticlesCreatePage />,
+        element: <RestrictedRoute allow={isArticleEditor} />,
```

Comment:
The articles API currently required admin permissions on the backend. `< RestrictedRoute />` restricts the access at render instead of after an API returns. (Which in the case of the article creation page, wasn't until a user submits a form.)